src/android/TP6mAggregate.jsx

```diff
@@ -342,7 +342,7 @@ class TP6mAggregate_ extends Component {
                           path: '/android/tp6m',
                           query: {
                             test,
-                            platform,
+                            platform: browserPlatform.id,
```

Comment:
Writing `platform: browserPlatform.id` means something is not named correctly:  ES6 re/destructuring promotes using a single name across functions and modules for the same thing.  When you see a name change, either something has the wrong name, or logic is wrong.  I am concerned that `platform` is different from `browserPlatform.id`.   I am guessing the problem you are trying to solve is somewhere else in the code.

* is `platform` value correct?  maybe it must be changed to `"fenix-*"`?

Reply:
@airimovici  a gentle reminder that this PR is still open.

Reply:
There is a confusion between platform and browserPlatform in several components and parts of the code. For this case in particular I suggested a fix below: to eliminate the confusion we should change the nav picker id from TP6m page from `platform` to `browserPlatform`